File: features_python/tmp.m
I = csvread('testimg.csv');
typeflag.transform = false;
typeflag.corr = false;
typeflag.global = false;
addpath('ImFEATbox/GlobalFeatures/Transformation/');
Out = DCTF(I, typeflag);
csvwrite('matlab-out.csv', Out);

File: features_python/tmp.py
import ImFEATbox, csv
import numpy as np
with open('testimg.csv', 'r') as csvfile:
	I = np.array(list(csv.reader(csvfile, delimiter=','))).astype(np.float)
typeflag = dict()
typeflag['corr'] = False
typeflag['texture'] = False
typeflag['global'] = True
typeflag['entropy'] = False
out = ImFEATbox.GlobalFeatures.Geometrical.glcm.cFeatures(I, typeflag)
with open('python-out.csv', 'wb') as csvfile:
	writer = csv.writer(csvfile, delimiter=',', escapechar=' ', quoting=csv.QUOTE_NONE)
	writer.writerow(out)